docs/ruleset_functions/expected_system_type_from_Table_G3_1_1.md

```diff
@@ -0,0 +1,117 @@
+# expected_system_type_from_Table_G3_1_1
+**Schema Version:** 0.0.22  
+
+**Description:** returns the expected system type based on the BAT, the number of floors of the building and the area of the BAT, and climate zone
+
+**Inputs:**
+- **building_area_type** - ("RESIDENTIAL", "PUBLIC_ASSEMBLY", "RETAIL", "HOSPITAL", "OTHER_NON_RESIDENTIAL", "HEATED-ONLY_STORAGE")
+- **number_of_floors** - this is the number of floors in the building integer greater than 0
+- **area_of_BAT** - this is the total area of the building area type
+
+**Returns:**  
+- **result**: a dict indicating the expected system type from table G3_1_1 ("SYS-1", "SYS-2", etc) and a string that indicates how the system was chosen, for example: {"EXPECTED_SYSTEM_TYPE": "SYS-4", "SYSTEM_ORIGIN": "PUBLIC_ASSEMBLY CZ_0_to_3a < 120,000 ft2"} - does not take into account G3.1.1 b-g
+ 
+**Function Call:**
+- **is_CZ_0_to_3a**
+
+## Logic:
+- create area_of_BAT_string, which will be appended to the details_of_system_selection string: `area_of_BAT_string = ""`
+- create number_of_floors_string, which will be appended to the details_of_system_selection string: `number_of_floors_string = ""`
+
+- get the climate zone category using the functions is_CZ_0_to_3a: `if is_CZ_0_to_3a():`
+	- set climate_zone_category to CZ_0_to_3a: `climate_zone_category = " CZ_0_to_3a"
```

Comment:
just a small comment, the function 'is_CZ_0_to_3a' results in a true or a false. So should line 21 be checking if 'is_CZ_0_to_3a' == TRUE?

Reply:
@supriyagoel - `if` evaluates a statement that should resolve to `true` or `false`, so when the function already returns `true` or `false`, you don't need the `== true`